app/views/users/show.html.erb

```diff
@@ -1 +1,13 @@
-<h1>Here is the info of a given user</h1>
\ No newline at end of file
+<%= render "shared/user", user: @user %>
+<div class="bio box">
+  <span class="title">Bio</span>
+  <span><%= @user.bio %></span>
+</div>
+<% @posts.each do |post| %>
+  <%= link_to user_post_path(@user, post) do %>
+    <%= render "shared/post", post: post %>
+  <% end %>
+<% end %>
+<div class="button-container">
+  <%= button_to "See all posts", user_posts_path(@user),:class => "posts-button", :method => :get %>
+</div>
```

Comment:
- [ ] When I click on this link I should be able to see a list of all user's posts but currently am getting this error please fix that
![link](https://user-images.githubusercontent.com/44745855/143121740-ef166412-6b77-4ef4-ab93-27b7177281d8.png)